notebooks/facloc-disperse-real-world.ipynb

```diff
@@ -556,18 +562,40 @@
     "        markeredgecolor=\"k\",\n",
     "        linewidth=0,\n",
     "        alpha=0.8,\n",
-    "        label=f\"y{fac_sites[i]} facility selected\",\n",
+    "        label=f\"{fname}\",\n",
     "    ))\n",
     "\n",
     "    plt.title(title, fontweight=\"bold\")\n",
-    "    plt.legend(handles = legend_elements, loc='upper left', bbox_to_anchor=(1.05, 1))"
+    "    plt.legend(handles = legend_elements, loc='upper left', bbox_to_anchor=(1.05, 1))\n",
+    "\n",
+    "    x, y, xyc, arrow_length, c = 0.925, 0.15, \"axes fraction\", 0.1 , \"center\"\n",
+    "    xy, xyt = (x, y), (x, y-arrow_length)\n",
+    "    ap = dict(facecolor=\"black\", width=5, headwidth=10)\n",
+    "    kws = dict(arrowprops=ap, ha=c, va=c, fontsize=20)\n",
+    "    plt.annotate(\"N\", xy=xy, xycoords=xyc, xytext=xyt, **kws)\n",
+    "    \n",
+    "    plt.gca().add_artist(ScaleBar(1))"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 103,
+   "execution_count": 136,
```

Comment:
Looks like you have an error going on [here](https://github.com/pysal/spopt/blob/df09ddc14f65f86c129eb4e68a250fc811d6fe51/notebooks/facloc-disperse-real-world.ipynb).